apps/web/composables/useInitalSetup/useInitialSetup.ts

```diff
@@ -49,6 +50,7 @@ const setInitialDataSSR: SetInitialData = async () => {
     setUser(data.value?.data.session as SessionResult);
     setCart(data.value?.data.session.basket as Cart);
     setCategoryTree(data.value.data.categories);
+    await fetchWishlist();
```

Comment:
I would suggest we load it without await. We can talk about it again after we implement the wishlist and where we want to show to button to access the list.